### DISCUSSION TITLE
Citing the mooseframework documentation and images

### DISCUSSION THREAD

**Discussion by MatthewWilliamNoble** [idaholab/moose#25534] [Q&A General] [answered]
Hello, 

I'm currently writing up my DPhil thesis. I am referencing the mooseframework documentation as my project utilised the MOOSE code. For the web pages, I have gone with a traditional `@online` block like so:

```
@online{mooseframework-FeatureFloodCount-2023,
	author = {Idaho National Lab.},
	title = {FeatureFloodCount},
	url = {https://mooseframework.inl.gov/source/postprocessors/FeatureFloodCount.html},
	urldate = {2023-09-11},
	year = {2023},
	bdsk-url-1 = {https://mooseframework.inl.gov/source/postprocessors/FeatureFloodCount.html}}
```

However, when describing the FeatureFloodCount class:

https://mooseframework.inl.gov/source/postprocessors/FeatureFloodCount.html

I was hoping to re-use the diagrams as a figure for the visual ease of describing what is going on.

Who would I need to ask for permission from to do this? Is this allowed at all?

Thank you, and all the best, 
Matthew

**Comment by GiudGiud** [comment]
@permcody worked on this

For citing MOOSE in general see this page
https://mooseframework.inl.gov/citing.html

**Reply by MatthewWilliamNoble** [reply]
Thank you for taking the time to reply, but that doesn't address my question. I am not citing the MOOSE codebase, what I am asking about is how to use the images in the MOOSE documentation in a PhD thesis. One cannot just copy and paste a picture or a plot or a photograph without permission. I am asking whose permission I would need to obtain to recreate the figure on the web page I linked explaining how the FeatureFloodCount class worked.

**Reply by GiudGiud** [reply]
@permcody worked on this particular feature of the code.
He can grant you permission, or he will know who to contact.

**Reply by permcody** [reply, marked as answer]
Hi @MatthewWilliamNoble, I don't see any issues with you using my images. Please do! They originally appears in my PhD thesis, but it may be easier to cite them with one of the follow-on papers: https://www.tandfonline.com/doi/full/10.1080/00295450.2020.1843893
